pkg/recovery/apiserver.go

```diff
@@ -334,6 +334,7 @@ func (s *Apiserver) Destroy() error {
 }
 
 func copyFile(src, dest string) error {
+	// #nosec
```

Comment:
I am skeptical to polluting code like this

Reply:
If we don't think this is a relevant check, we can disable it so we don't need to add these flags in this case.